app/entry.js

```diff
@@ -3,14 +3,17 @@
 // webpack assets
 require('!!file?name=[name].[ext]!./html/index.html');
```

Comment:
this entery point is very well organized, it reads very well from top to bottom,
It make it very easy to know what this app does, and where to find a feature that needs work